### PR TITLE
uhttpd: update to latest version

### PR DESCRIPTION
**Pull request by ianchi**
The version pointed to by Lede's package was many commits behind HEAD of uHttpd project.
Bump to latest.

### CODE REVIEW

**Comment by diizzyy** [comment]
Update pkg_mirror_hash and fix the typo (lastest)? :-)

**Comment by ianchi** [comment]
@diizzyy  
updated, thanks for the catch.

File: package/network/services/uhttpd/Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
 PKG_NAME:=uhttpd
-PKG_RELEASE:=2
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(LEDE_GIT)/project/uhttpd.git
-PKG_SOURCE_DATE:=2016-10-25
-PKG_SOURCE_VERSION:=1628fa4b34aa143187353f81e8001b9a15286bda
-PKG_MIRROR_HASH:=2ac4ba8dc0b349d72174aac9ff693a73a214295a9890fe3d2a8eedcad54d06e3
+PKG_SOURCE_DATE:=2017-07-21
```
Comment (stintel) [inline package/network/services/uhttpd/Makefile]: Use commit date, not author date.

**Comment by ianchi** [comment]
Updated to commit date and added shortlog to commit message.

**Comment by ianchi** [comment]
@stintel / @diizzyy, any additional comments?

**Comment by diizzyy** [comment]
LGTM

**Comment by dedeckeh** [comment]
Thanks for the patch; merged into master

**Pull request closed without merging**